### PR TITLE
docs: update changelog for 1.18.0

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Update changelog for 1.18.0

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,61 @@
+CHANGES IN VERSION 1.18.0 (FROM 1.17.1)
+===================================
+
+Appium 1.18.0 is a patch release
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:4]: patch or minor?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,61 @@
...
+
+#### Android General
+* Add capabilities:
+  * `mockLocationApp` to make location mock configurable [appium-android-driver#632](https://github.com/appium/appium-android-driver/pull/632)
+  * `logcatFormat`, `logcatFilterSpecs` to allow to customise logcat format [appium-adb#528](https://github.com/appium/appium-adb/pull/528)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:9]: to allow logcat output format customization

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,61 @@
...
+  * `ignoreHiddenApiPolicyError` to ignore permission error when hidden api policy change happenes [appium-adb#507](https://github.com/appium/appium-adb/pull/507)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:10]: happens

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,61 @@
...
+
+#### Android (Espresso)
+* Add capabilities:
+  * `disableSuppressAccessibilityService` to control [FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES]((https://developer.android.com/reference/android/app/UiAutomation#FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES)) flag [appium-espresso-driver#559](https://github.com/appium/appium-espresso-driver/pull/559)
+  * `appLocale` to set [Locale](https://developer.android.com/reference/java/util/Locale) for [the target context](https://developer.android.com/reference/androidx/test/core/app/ApplicationProvider#getApplicationContext()) [appium-espresso-driver#580](https://github.com/appium/appium-espresso-driver/pull/580)
+* Add `mobile:` functions:
+  * `mobile: sensorSet` to set sensor [appium-espresso-driver](https://github.com/appium/appium-espresso-driver/pull/560)
+
+### iOS (XCUITest)
+* Add for Xcode 12 beta support
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:38]: `for` is not needed

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,61 @@
+CHANGES IN VERSION 1.18.0 (FROM 1.17.1)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:1]: important fixes that affect all drivers: 
- https://github.com/appium/appium-base-driver/commit/d946ba1846b5dd43c5ce59b244cf4eb5b55c7369
- https://github.com/appium/appium-base-driver/commit/9015d4c80d12d2ed62cd3f96b3389963f1d43349

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,68 @@
...
+  * `boundElementsByIndex` for bounding strategy to lookup elements [WebDriverAgent#357](https://github.com/appium/WebDriverAgent/pull/357)
+* Add XCTest `mobile:` functions: [appium-xcuitest-driver#1205](https://github.com/appium/appium-xcuitest-driver/pull/1205). Please read [test code](https://github.com/appium/appium-xcuitest-driver/blob/master/test/functional/device/xctest-e2e-specs.js) as an example.
+  * `mobile: runXCTest` to run XCTest bundle
+  * `mobile: installXCTestBundle` to install the xctest bundle
+  * `mobile: listXCTestBundles` to get list of xctests within bundle
+  * `mobile: listXCTestsInTestBundle` to return the list of XCTest bundles
+* Add audio recording for simulators and real devices [appium-xcuitest-driver#1207](https://github.com/appium/appium-xcuitest-driver/pull/1207)
+  * Read [Audio Capture From iOS Simulators and Real Devices](https://appium.io/docs/en/writing-running-appium/ios/audio-capture/)
+* Fix removing unnecessary dependencies [WebDriverAgent#348](https://github.com/appium/WebDriverAgent/pull/348) [WebDriverAgent#350](https://github.com/appium/WebDriverAgent/pull/350) [#351](https://github.com/appium/WebDriverAgent/pull/351)
```
Comment (dpgraham) [inline CHANGELOG.md:56]: I think we can exclude this because it doesn't effect the user.

Comment (mykola-mokhnach) [reply CHANGELOG.md]: it does affect the user in a way there was a memory leak while the old deps were used. Now the leak is fixed

Comment (KazuCocoa) [reply CHANGELOG.md]: I should have addressed the memory leak to clarify the reason. Did in b956b5c

**Review by mykola-mokhnach — APPROVED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,68 @@
+CHANGES IN VERSION 1.18.0 (FROM 1.17.1)
+===================================
+
+Appium 1.18.0 is a minor release
+
+#### General
+* Add `score` attribute for ImageElement [appium-base-driver#396](https://github.com/appium/appium-base-driver/pull/396)
+* Add to proxy Chrome DevTools endpoint to downstream drivers [appium-base-driver#405](https://github.com/appium/appium-base-driver/pull/405)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:8]: Add a route to be able to execute Chrome DevTools commands  ...

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,68 @@
...
+  * e.g. Clients can send `/session/:sessionId/goog/cdp/execute` command to Chromedriver
+* Fix socket leak by handling connections in a shared pool [appium-base-driver#416](https://github.com/appium/appium-base-driver/pull/416)
+* Fix to encode filenames with UTF-8 in extracting files from `.ipa` [appium-base-driver#419](https://github.com/appium/appium-base-driver/pull/419)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:11]: ... while extracting `.ipa` packages

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,68 @@
...
+  * This change fixes installing `.ipa` which bundles UTF-8 file names such as [this issue](https://github.com/appium/appium/issues/14100)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:12]: The change fixes `.ipa` packages deployment that contain file names containing non-ASCII characters

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,68 @@
...
+* Add XCTest `mobile:` functions: [appium-xcuitest-driver#1205](https://github.com/appium/appium-xcuitest-driver/pull/1205). Please read [test code](https://github.com/appium/appium-xcuitest-driver/blob/master/test/functional/device/xctest-e2e-specs.js) as an example.
+  * `mobile: runXCTest` to run XCTest bundle
+  * `mobile: installXCTestBundle` to install the xctest bundle
+  * `mobile: listXCTestBundles` to get list of xctests within bundle
+  * `mobile: listXCTestsInTestBundle` to return the list of XCTest bundles
+* Add audio recording for simulators and real devices [appium-xcuitest-driver#1207](https://github.com/appium/appium-xcuitest-driver/pull/1207)
+  * Read [Audio Capture From iOS Simulators and Real Devices](https://appium.io/docs/en/writing-running-appium/ios/audio-capture/)
+* Fix a memory leak with removing unnecessary dependencies [WebDriverAgent#348](https://github.com/appium/WebDriverAgent/pull/348) [WebDriverAgent#350](https://github.com/appium/WebDriverAgent/pull/350) [WebDriverAgent#351](https://github.com/appium/WebDriverAgent/pull/351)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:56]: ...by removing...

**Comment by KazuCocoa** [comment]
ping @dpgraham

**Pull request merged**